OpenOversight/tests/test_database_cache.py

```diff
@@ -71,6 +75,8 @@ def test_documented_assignments(mockdata, client, faker):
     with current_app.test_request_context():
         login_admin(client)
         department = Department.query.first()
+        assert isinstance(department, Department)
+
```

Comment:
Is this still needed?

Reply:
This was entered to some degree to dissuade the linter, but I'm happy to remove it.